src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_util.py

```diff
@@ -0,0 +1,194 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import register_cli_argument, cli_command
+from azure.cli.core.commands.parameters import ignore_type
+from azure.cli.core.commands.arm import cli_generic_update_command
+
+
+# MANAGEMENT CLIENT FACTORIES
```

Comment:
These should go in a _client_factory.py file to be consistent with other packages.